### PR TITLE
refactor kubelet, remove useless parameters

### PR DESCRIPTION
**Pull request by oulinbao**
**What this PR does / why we need it**:
remove useless parameter in NewMainKubelet func

**Special notes for your reviewer**:

**Release note**:
```release-note
None
```

### CODE REVIEW

**Comment by dixudx** [comment]
/ok-to-test

@mtaufen @wackxu This pr is better, just like what I said in [#56616](https://github.com/kubernetes/kubernetes/pull/56616#discussion_r154010859).

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -203,7 +203,6 @@ type Builder func(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	registerNode bool,
 	registerWithTaints []api.Taint,
 	allowedUnsafeSysctls []string,
-	containerized bool,
```
Comment (mtaufen) [inline pkg/kubelet/kubelet.go]: You should also remove it from the Kubelet struct in this file.

Comment (oulinbao) [reply pkg/kubelet/kubelet.go]: OK, I have removed it.

**Comment by oulinbao** [comment]
/retest




发自网易邮箱大师
On 12/23/2017 08:13, k8s-ci-robot wrote:

@oulinbao: The following test failed, say /retest to rerun them all:

| Test name | Commit | Details | Rerun command |
| pull-kubernetes-unit | 1da1291 | link | /test pull-kubernetes-unit |

Full PR test history. Your PR dashboard. Please help us cut down on flakes by linking to an open issue when you hit one in your PR.

Instructions for interacting with me using PR comments are available here. If you have questions or suggestions related to my behavior, please file an issue against the kubernetes/test-infra repository. I understand the commands that are listed here.

**Pull request closed without merging**